igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>I think it'd be worth adding a summary of the Edgecumbe story first thing to give the "why" you would run these analyses. </p><p> </p><p>I actually like the benchmarks and might fold it into the why blurb. Something like "Using OpenSARLab and HyP3, Grapenthin et al. processed __ GB S1 SLC data into __ interferrograms totaling __ GB and then analyzed the time series in MIntPy resulting in an additional __ GB analysis data. We'll show you how to do the same in a matter of hours."  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/tutorials/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>